standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Runtime.xml

```diff
@@ -20091,11 +20091,8 @@ The <see cref="P:System.Nullable`1.HasValue" /> property for the current <see cr
       <returns>The hash code.</returns>
     </member>
     <member name="M:System.Range.GetOffsetAndLength(System.Int32)">
-      <summary>Returns a <see cref="T:System.Range.OffsetAndLength" /> instance with the starting offset and length of a range.</summary>
-      <param name="length">A positive integer that represents the length of the collection that the range will be used with.</param>
-      <returns>The starting offset and length of the range.</returns>
-      <exception cref="T:System.ArgumentOutOfRangeException">
-        <paramref name="length" /> is outside the bounds of the current range.</exception>
+      <summary>Calculates the start offset and length of the range object using a collection length.</summary>
+      <param name="length">The length of the collection that the range will be used with. <paramref name="length" /> has to be a positive value.</param>
```

Comment:
Looks like we are losing some content here (like return tag/exception):
https://github.com/dotnet/corefx/blob/50e8db3cabbfb6ad0fc0702b3db25491a95c72a5/src/Common/src/CoreLib/System/Range.cs#L92-L100

I think what's checked in missed some details and it was removed in this PR:
https://github.com/dotnet/dotnet-api-docs/pull/3055/files#diff-329e2957d7afa212df1c2d8a57ae415aL278

https://docs.microsoft.com/en-us/dotnet/api/system.range.getoffsetandlength?view=netcore-3.0

Reply:
I'll investigate.

Reply:
Ok but this is another example of the intellisense matching docs.

Reply:
yes, but this was a problem with the CI. I'll add a suggestion to add the two missing sections and open a PR in docs too.